gui/wxpython/datacatalog/tree.py

```diff
@@ -294,6 +294,15 @@ def __init__(
             self.grassdatabases.append(currentDB)
             self._saveGrassDBs()
 
+        self.ReloadTreeItems()
+
+        # If demolocation, show the map world layer in the Map Display
+        location = gisenv()['LOCATION_NAME']
+        mapset = gisenv()['MAPSET']
+        if (location == "world_latlong_wgs84"
+        and mapset == "PERMANENT"):
+            self.DefineDemolocation('country_boundaries', 'vector')
```

Comment:
Why adding of a layer should be part data catalog? I'm not even convinced about all first-time user info bars. In any case, whether displaying layer is a display issue or a state of the whole GUI, these should be primarily solved in lmgr, so try it there first.